transforms/language/html2parquet/notebooks/html2parquet.ipynb

```diff
@@ -0,0 +1,220 @@
+      "17:09:47 INFO - Completed 1 files (100.0%) in 0.111 min\n",
+      "17:09:47 INFO - Done processing 1 files, waiting for flush() completion.\n",
+      "17:09:47 INFO - done flushing in 0.0 sec\n",
+      "17:09:47 INFO - Completed execution in 0.111 min, execution result 0\n"
+     ]
+    }
+   ],
+   "source": [
+    "\n",
+    "import sys\n",
+    "sys.argv = ParamsUtils.dict_to_req(d=(params))\n",
+    "# create launcher\n",
+    "launcher = PythonTransformLauncher(Html2ParquetPythonTransformConfiguration())\n",
+    "# launch\n",
+    "return_code = launcher.launch()\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "e2bee8da-c566-4e45-bca1-354dfd04b0df",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>title</th>\n",
+       "      <th>document</th>\n",
+       "      <th>contents</th>\n",
+       "      <th>document_id</th>\n",
+       "      <th>size</th>\n",
+       "      <th>date_acquired</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>ai-alliance-index.html</td>\n",
+       "      <td>ai-alliance-index.html</td>\n",
+       "      <td>![](https://images.prismic.io/ai-alliance/Ztf3...</td>\n",
+       "      <td>f86b8cebe07ec9f43a351bb4dc897f162f5a88cbb0f121...</td>\n",
+       "      <td>394</td>\n",
+       "      <td>2024-11-13T17:09:40.947095</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                    title                document  \\\n",
+       "0  ai-alliance-index.html  ai-alliance-index.html   \n",
+       "\n",
+       "                                            contents  \\\n",
+       "0  ![](https://images.prismic.io/ai-alliance/Ztf3...   \n",
+       "\n",
+       "                                         document_id  size  \\\n",
+       "0  f86b8cebe07ec9f43a351bb4dc897f162f5a88cbb0f121...   394   \n",
+       "\n",
+       "                date_acquired  \n",
+       "0  2024-11-13T17:09:40.947095  "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pyarrow.parquet as pq\n",
+    "import pandas as pd\n",
+    "table = pq.read_table('output/ai-alliance-index.parquet')\n",
```

Comment:
table = pq.read_table('/path/to/your/output/folder/sample.parquet')

users don't have "ai-alliance-index.parquet" so making it general.